include.js

```diff
@@ -6,6 +6,13 @@ let cards = [
     author: 'Krish Gautam',
     githubLink: 'https://github.com/Kris248'
   },
+  {
+    artName: 'lightning square',
+    pageLink: './Art/aman/index.html',
+    imageLink: './Art/aman/Screencast from 2023-10-24 17-49-04.webm',
```

Comment:
Thanks for this contribution.

Please check the following checklist that you need to fulfill for us to be able to merge your request.

Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [ ] the contribution folder in `./Art` is the contributor's Github name
- [ ] has a gif or png file of the animation
- [x] can find the html 
- [x] can find the style (css file or inline)
- [ ] modified `include.js` **with the right details**

## Issues: 

- please rename the folder of your contribution so that it's the same as your Github username
- please remove the `.webm` file and replace it with a `.png` or `.gif`
- please make sure to adjust your entry in `include.js` at the end.

If you have questions feel free to reach out on the discord channel #hacktoberfest-2023 or ask them here in this pull request.